easybuild/tools/filetools.py

```diff
@@ -2473,3 +2473,43 @@ def copy_framework_files(paths, target_dir):
             raise EasyBuildError("Couldn't find parent folder of updated file: %s", path)
 
     return file_info
+
+
+def create_unused_dir(parent_folder, name):
+    """
+    Create a new folder in parent_folder using name as the name.
+    When a folder of that name already exists, '_0' is appended which is retried for increasing numbers until
+    an unused name was found
+    """
+    if not os.path.isabs(parent_folder):
+        parent_folder = os.path.abspath(parent_folder)
+
+    start_path = os.path.join(parent_folder, name)
+    number = None
+    while True:
```

Comment:
This `while True` makes me a bit nervous...

How about giving up after say 1000 attempts? So use `for _ in range(1000):` rather than `while True`?

Reply:
Ok, done